api/serverless.yml

```diff
@@ -43,7 +43,7 @@ provider:
 
   environment:
     AWS_NODEJS_CONNECTION_REUSE_ENABLED: 1
-    BOS_JWT_SECRET: ${ssm:/aws/reference/secretsmanager/BOS_JWT_SECRET}
+    BOS_JWT_SECRET: ${ssm:/aws/reference/secretsmanager/BOS_JWT_SECRET~true}
```

Comment:
`~true` is an optional (not so optional for us) flag telling serverless framework to decrypt the secret